tutorials/lab/ssh.rst

```diff
@@ -0,0 +1,99 @@
+.. _ssh-between-clusters:
+
+====================
+SSH between clusters
+====================
+
+Let's say you want to create a Flux instance in an allocation on a cluster (e.g., lassen)
+and then connect to it via ssh from another cluster (quartz). This is possible with the right
+setup of your ``~/.ssh/config``. 
+
+----------------------
+Create a Flux Instance
+----------------------
+
+
+First, let's create the allocation on the first cluster.
```

Comment:
Would it be possible to generalize it a bit?  On clusters with native flux you might typically do  ... on clusters with slurm ... on lassen see <link>?

Reply:
@grondo this command?

```
$ jsrun -a 1 -c ALL_CPUS -g ALL_GPUS -n ${NUM_NODES} --bind=none --smpiargs="-disable_gpu_hooks" flux start
```
This definitely calls for a "Rosetta stone" of starting flux commands - e.g., I could imagine the same but for slurm, or another job manager. I think there should be one place where someone can go and see them all side by side (although separate from this PR). What I can do is put this link / reference in the tutorial, and I'm still thinking about how we can structure a more general set of mini tutorials.

Reply:
@chu11 I agree we want this "how to start flux" across different places somewhere, maybe not here but definitely somewhere, along with other useful commands and different contexts / quick reference for running them.

Reply:
i thought we already had the how to launch a sub-instance on FAQ, but I guess not ... seems like we should definitely add something.

Reply:
@grondo I'm testing jsrun, and maybe I'm doing it wrong, but this doesn't work:

```
$ export NUM_NODES=1
$ jsrun -a 1 -c ALL_CPUS -g ALL_GPUS -n ${NUM_NODES} --bind=none --smpiargs="-disable_gpu_hooks" flux start
Error: It is only possible to use js commands within a job allocation unless CSM is running
02-06-2023 13:43:54:480 82493 main: Error initializing RM connection. Exiting.
```
So I followed the message and got an allocation first:

```bash
$ jsrun -a 1 -c ALL_CPUS -g ALL_GPUS -n ${NUM_NODES} --bind=none --smpiargs="-disable_gpu_hooks" flux start
flux-broker: stdin is not a tty - can't run interactive shell
```
(and no flux instance is started) is there another command on that page I should be using? https://flux-framework.readthedocs.io/en/latest/tutorials/lab/coral.html#launching-spectrum-mpi-within-flux

Reply:
@chu11 I'm trying to get that working right now (see above) no luck so far! I agree we need an entire general page on this too.

Reply:
Ping @grondo I need help with the jsrun command (doesn't seem to work) when you have a moment! I think now #194 is waiting on this to merge first.

Reply:
for the purposes of this example, I don't think it needs to be jsrun, right?  Could it just be replaced with a slurm example?

Reply:
It definitely could, I just don't know what that looks like, haha. I wanted to test it too to ensure it works, but if someone else knows it works for sure I'd be happy to add that!

I agree since we generalized it slurm would be better, I don't know what a jsrun is :)

Reply:
This is from the lab's tutorial that @ryanday36 wrote.

```
>salloc -N4 --exclusive
salloc: Granted job allocation 321075
salloc: Waiting for resource configuration
salloc: Nodes opal[63-66] are ready for job
>srun -N4 -n4 --pty --mpibind=off flux start
>flux mini run -N4 hostname
opal63
opal64
opal65
opal66
```

Reply:
Another way from a manpage is: `srun --pty -N8 flux start`

I'm actually struggling to find a way to do it A) without a pty and B) allow the user to easily get the FLUX_URI ...

Reply:
Ah I see - so it's getting an allocation first still, but then running flux start with srun and then you can interact. Let me update the tutorial to use that, and if I can get access to a slurm cluster sometime with flux I can test again.

Reply:
> I'm actually struggling to find a way to do it A) without a pty and B) allow the user to easily get the FLUX_URI ...

Ahh `flux uri` has a slurm resolver based on a searching pids.

Reply:
I don't know `jsrun` usage either. You can also connect directly to a flux instance running under Slurm with `flux proxy slurm:JOBID`. This is documented in [Working with Flux job hierarchies](https://flux-framework.readthedocs.io/en/latest/jobs/hierarchies.html)